src/components/Queens/Queens.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Queen from './Queen.jsx';
+import { Link } from 'react-router-dom';
+
+const Queens = ({ queens }) => {
+  const queenElements = queens.map(queen => (
+    <Link to={`/Detail/${queen.id}`} key={queen.id}>
+      <li>
+        <Queen {...queen} />
+      </li>
+    </Link>
+  ));
+
+  return (
+    <ul>
+      {queenElements}
+    </ul>
+  );
+};
+
+Queens.propTypes = {
+  queens: PropTypes.arrayOf(PropTypes.shape({
+    id: PropTypes.number.isRequired,
```

Comment:
I don't think you need this line since you're not using the id you're getting from your fetch call but if you do still want to leave it there I would get rid of the `.isRequired` at the end so you don't have a warning in your console about it.